src/main/java/lotto/view/OutputView.java

```diff
@@ -23,11 +23,11 @@ public static void printPurchaseQuantity(long quantity) {
 	public static void printWinningStatistics(RankingResult rankingResult) {
 		StringBuffer printBuffer = new StringBuffer();
 
-		printBuffer.append("당첨 통계");
-		printBuffer.append("---------");
+		printBuffer.append("당첨 통계\n");
```

Comment:
```suggestion
		printBuffer.append("당첨 통계").append(ENTER);
```
이런식으로 명확하게 분리해 보면 좀 더 가독성에 좋을 것 같습니다 😄

Reply:
\n을 상수로 뺀다는 생각을 못했네요! 훨씬 깔끔해졌네요! 👍